src/rest/Places/util.ts

```diff
@@ -0,0 +1,22 @@
+import type { Request, Response } from "express";
+import { getPlaceByID } from "../../db/Place/place";
+import { DbOperationError } from "../../errorClasses";
+import { handleError } from "../util";
+
+export const placeExists = async <T>(
+	placeID: string,
+	endPointName: string,
+	req: Request<unknown, unknown, T>,
+	res: Response
+): Promise<boolean> => {
+	try {
+		await getPlaceByID(placeID);
+	} catch (e) {
+		if (e instanceof DbOperationError) {
+			return false;
+		} else {
+			handleError<T>(e, endPointName, req, res);
+		}
+	}
+	return true;
+};
```

Comment:
I don't quite understand the purpose of this function - we have functionality built in to getPlaceByID to create a new place if it doesn't exist already and return that - so i think this would always return true, unless there's an error, in which case just using getPlaceByID would be throwing the same error as this function

Reply:
lmk if i'm wrong, not tryna block merging over something pretty trivial

Reply:
the difference between just calling `getPlaceByID` and this function is that this function calls `handleError`. else, we would have this exact structure peppered among several of the rest endpoints, which violates DRY.